lib/foods/foods.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function() {
+  $(loadFoods)
+})
+
+function loadFoods() {
+  $.get('https://obscure-harbor-85447.herokuapp.com/api/v1/foods')
+  .then(function(foods) {
+    foods.forEach(function(food) {
+      $('.foods-table').append(
+        `<tr class=row${food.id}><td>`+ food.name + '</td>' +
+        '<td>' + food.calories + '</td>' +
+        `<td><input id=${food.id} type='button' class='delete-row' value='Delete'/></td>` +
+        "</tr>"
+      )
+    })
+  })
+}
+
+function deleteFood(e) {
+  $.ajax({
+    type: 'DELETE',
+    url:'https://obscure-harbor-85447.herokuapp.com/api/v1/foods/' + e.target.id,
```

Comment:
it looks like you're reusing the base of this url. can we extract it out and make it a constant so we can reuse it? 👍